R/import.R

```diff
@@ -0,0 +1,50 @@
+#' @importFrom magrittr "%>%"
+#' @importFrom pkgmaker "sVariable"
+#' @importFrom purrr "map"
+#' @importFrom modules "import_"
+
+
+importsStack <- pkgmaker::sVariable(c("."))
+
+shortPath <- function(path) {
```

Comment:
what this function does is unclear to me. I think it needs a comment and rewriting the body to be readable and straightforward